### PR TITLE
Tarea 11

### PR DESCRIPTION
**Pull request by santiagoGuillenGlob**

### CODE REVIEW

File: santiao-guillen/homework-11/to-do-list-router/src/Components/Homework2.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+const Homework2 = () => {
```
Comment (emolinaglobant) [inline santiao-guillen/homework-11/to-do-list-router/src/Components/Homework2.js]: @santiagoGuillenGlob , this approach is really wrong, your basically repeating the code in different components, you should use only one Homework component and pass props with the info to it

File: santiao-guillen/homework-11/to-do-list-router/src/Components/Navegation.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import Homework5 from './Homework5';
+
+import '../App.css';
+
+const Navegation = () => {
+    
+    return (
+        <Router>
+            <nav className="container">
+                <ul>
```
Comment (emolinaglobant) [inline santiao-guillen/homework-11/to-do-list-router/src/Components/Navegation.js]: @santiagoGuillenGlob , this list should be dynamic and come from an Array

File: santiao-guillen/homework-11/to-do-list-router/src/Components/Navegation.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                        <Link className="homework" to="/Homework-4">🧾Homework</Link>
+                    </li>
+                    <li>
+                        <Link className="homework" to="/Homework-5">🧾Homework</Link>
+                    </li>
+                </ul>
+            </nav>
+
+            <Routes>
+                <Route path="/Homework-1" element={<Homework1 />} />
```
Comment (emolinaglobant) [inline santiao-guillen/homework-11/to-do-list-router/src/Components/Navegation.js]: @santiagoGuillenGlob , same here

**Review by emolinaglobant — CHANGES_REQUESTED** [review]

**Comment by emolinaglobant** [comment]
@santiagoGuillenGlob the task list is not working

**Comment by santiagoGuillenGlob** [comment]
because i used the node server for a create a fake api, where a stay save the task, You can run the server whit npm run server

**Comment by santiagoGuillenGlob** [comment]
This is error?

**Pull request closed without merging**